packages/custom-functions-metadata/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "custom-functions-metadata",
-  "version": "1.5.5",
+  "version": "1.5.6",
```

Comment:
We shouldn't change the version manually . . . that is handled by Lerna when we publish the packages.